__tests__/moment_spec.re

```diff
@@ -740,6 +740,10 @@ let () =
           expect(moment("2017-01-02 21:00:00+11:00") |> Moment.tz)
           |> toBeUndefined
         );
+        test("#get utc offset", () =>
```

Comment:
Minor: the other tests use the function name as the test name, which means this should be, `utcOffset`.